app/src/main/res/values/strings.xml

```diff
@@ -460,7 +460,7 @@
         <item>Medium</item>
         <item>Large</item>
     </string-array>
-    <string name="style_choose_title">App theme</string>
+    <string name="style_choose_title">Theme</string>
```

Comment:
Subsequently, there could be different themes for the databases.

Reply:
What would a database theme be? Is that possible?

Reply:
There are colors set in the XML format of databases. I will reuse these variables later to make themes specific to each database. You can see these colors in the options of KeePass programs for PC (even if for the moment nothing changes visually)